draft-ietf-quic-transport.md

```diff
@@ -4085,6 +4085,13 @@ UDP datagrams MUST NOT be fragmented at the IP layer.  In IPv4
 ({{!IPv4=RFC0791}}), the DF bit MUST be set if possible, to prevent
 fragmentation on the path.
 
+Even though datagrams with certain properties are required to be padded, the
+size of the datagram is not authenticated, and endpoints might send coalesced
+packets after the handshake is confirmed (see {{packet-coalesce}}).  Therefore,
```

Comment:
```suggestion
size of the datagram is not authenticated.  Therefore,
```
I don't think that we need to call this out, but you can use the other suggestion if you prefer.

Reply:
Thank you for the review. I removed the callout in 353747d.